locales-zh-TW.xml

```diff
@@ -148,6 +148,7 @@
 
     <!-- VERB ROLE FORMS -->
     <term name="author" form="verb">著</term>
```

Comment:
No clue. I would assume we would hear an outcry from Japanese users, though, since we use this term a lot in "author" macros that have substitution with the "editor" and/or "translator" variables (in which case the term is usually desirable).